File: gluecodium/src/test/resources/smoke/internal_fields/output/android/com/example/smoke/PublicFieldsNoInit.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class PublicFieldsNoInit {
    @NonNull
    public String publicField;
    @NonNull
    String internalField;
    public PublicFieldsNoInit(@NonNull final String publicField) {
        this.publicField = publicField;
        this.internalField = "foo";
    }
}

File: gluecodium/src/test/resources/smoke/internal_fields/output/android-kotlin/com/example/smoke/PublicFieldsNoInit.kt
/*

 *
 */

@file:JvmName("PublicFieldsNoInitExtensions")

package com.example.smoke


class PublicFieldsNoInit {
    @JvmField var publicField: String
    @JvmField internal var internalField: String



    constructor(publicField: String) {
        this.publicField = publicField
        this.internalField = "foo"
    }





}

